packages/api/src/functions/dbAuth/DbAuthHandler.ts

```diff
@@ -322,10 +322,16 @@ export class DbAuthHandler {
           `Username is required`
       )
     }
+    let user
 
-    let user = await this.dbAccessor.findUnique({
-      where: { [this.options.authFields.username]: username },
-    })
+    try {
+      user = await this.dbAccessor.findUnique({
+        where: { [this.options.authFields.username]: username },
+      })
+    } catch (e) {
+      console.log(e)
```

Comment:
Might you consider using the Redwood logger here?

Reply:
Thanks for the feedback @dthyresson! In your [initial issue](https://github.com/redwoodjs/redwood/issues/4679), you mentioned:
>  log the original error -- but no access to api logger here, so would have to simply console out

That is why I went with `console.log`. I could import `createLogger` and create a new instance of a logger that will be the default logger. 

Do you think the latter option is better?